recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -325,34 +341,53 @@ def galc(w):
     logger.info(msg)
 
 
-def check_commonvoice_folders(data_folder):
-    """
-    Check if the data folder actually contains the Common Voice dataset.
 
-    If not, raises an error.
 
+def check_commonvoice_folders(data_folder,dataset_version,language):
```

Comment:
If we choose to not use `datasets` then this function will have to be modified.

Reply:
I change the download dataset function and remove the dependencies to dataset. Please check download_locale function is common_voice_prepare.py